src/components/users/Profile.js

```diff
@@ -47,11 +48,15 @@ export default class Login extends Component {
       display: 'inline-block',
     };
 
+    var player = storeUtil.getPlayer(this.props.user.playerId);
+
     return (
       <Paper zDepth={1} style={paperStyle}>
         <h3>{this.context.t('profile.headerText')}</h3>
 
-        <p>{this.context.t('profile.loggedInText')}&nbsp;{this._reverseName((storeUtil.getPlayer(this.props.user.playerId) || {name: 'Broken Code'}).name)}</p>
+        <p>{this.context.t('profile.loggedInText')}&nbsp;{this._reverseName((player || {name: 'Broken Code'}).name)}</p>
+        <p>{this.context.t('profile.email')}&nbsp;{player.email}</p>
```

Comment:
Ik dacht het voorlopig te tonen zodat iemand het kan melden aan ons als zijn/haar info verkeerd is of er nog niet is :)